config/seata-config-core/src/test/java/org/apache/seata/config/FileConfigurationTest.java

```diff
@@ -41,13 +41,17 @@ void addConfigListener() throws InterruptedException {
         Configuration fileConfig = ConfigurationFactory.getInstance();
         CountDownLatch countDownLatch = new CountDownLatch(1);
         boolean value = fileConfig.getBoolean("service.disableGlobalTransaction");
-        ConfigurationCache.addConfigListener("service.disableGlobalTransaction", (event) -> {
-            Assertions.assertEquals(Boolean.parseBoolean(event.getNewValue()), !Boolean.parseBoolean(event.getOldValue()));
-            countDownLatch.countDown();
+        fileConfig.addConfigListener("service.disableGlobalTransaction", new CachedConfigurationChangeListener() {
+            @Override
+            public void onChangeEvent(ConfigurationChangeEvent event) {
+                Assertions.assertEquals(Boolean.parseBoolean(event.getNewValue()), !Boolean.parseBoolean(event.getOldValue()));
+                countDownLatch.countDown();
+            }
         });
         System.setProperty("service.disableGlobalTransaction", String.valueOf(!value));
         countDownLatch.await(5, TimeUnit.SECONDS);
         System.setProperty("file.listener.enabled", "false");
+        Thread.sleep(1000);
```

Comment:
Why sleep for 1 second?

Reply:
For thread safety, the following sleep of 2 seconds is meaningless.

Reply:
ok